.github/workflows/ci.yml

```diff
@@ -27,87 +25,48 @@ jobs:
         DATABASE_PORT: 3306
         SECRET_KEY: ${{ secrets.SECRET_KEY }}
         DJANGO_SETTINGS_MODULE: answerking.settings.development
-
     steps:
-    - uses: actions/checkout@v3
-
-    - name: Install poetry
-      run: pipx install poetry
-
-    - name: Set up Python 3.10
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.10'
-        cache: 'poetry'
-
-    - run: poetry install
+      - uses: ./.github/actions/dependencies
```

Comment:
This should be the first thing you run in the steps part of every job "- uses: actions/checkout@v3"

get rid of -uses:actions/dependancies